src/utils/execPath.ts

```diff
@@ -12,7 +12,10 @@ export function getAnsibleCommandExecPath(
   name: string,
   settings: ExtensionSettings
 ): string {
-  return name === 'ansible-lint'
-    ? settings.ansibleLint.path
-    : path.join(path.dirname(settings.ansible.path), name);
+  if (settings.ansible?.path) {
```

Comment:
Same here: `ansible.path` is not optional according to the interface. Hence expecting it to be null is against the compiler.